assets/clp_presto/Dockerfile

```
@@ -0,0 +1,77 @@
+RUN cd /usr/local && sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -d;
+
+RUN pip3 install pyarrow;
+
+WORKDIR /home
+RUN git clone https://github.com/anlowee/presto.git;
+WORKDIR /home/presto
+# RUN git checkout xwei/benchmark-test;
+RUN git checkout xwei/e2e-test-07-08;
+WORKDIR /home/presto/presto-native-execution
+RUN mkdir build;
+RUN rm -rf velox;
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)